numpy/add_newdocs.py

```diff
@@ -668,9 +668,8 @@ def luf(lamdaexpr, *args, **kwargs):
         will be in C-contiguous order (last-index varies the fastest).
         If order is 'F', then the returned array will be in
         Fortran-contiguous order (first-index varies the fastest).
-        If order is 'A' (default), then the returned array may be
-        in any order (either C-, Fortran-contiguous, or even discontiguous),
-        unless a copy is required, in which case it will be C-contiguous.
+        If order is 'A' (default), then the order of the returned array is
```

Comment:
There is a problem with a double meaning for A, the  "C if not F" meaning and "Anything" meaning, which I guess was the reason to add "K" at the time.